### PR TITLE
fix: double calling redeemcode function

### PR DESCRIPTION
**Pull request by sunilsabatp**
Fixes 

visit to this [URL](https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1742)  to see the issue

### CODE REVIEW

**Comment by norbertschuler** [comment]
Is this the same or a different issue as https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1707?

**Comment by sunilsabatp** [comment]
> Is this the same or a different issue as #1707?

it is an different issue, not related to button(`multiple click`)

**Comment by mariahosfeld** [comment]
Functionality looks good.

File: pages/profile/redeem/[code].tsx [hunk 1]
```diff
@@ -103,16 +103,15 @@ const ReedemCode: FC = () => {
...
 
   const changeRouteCode = () => {
     router.push(`/profile/redeem/${inputCode}?inputCode=${false}`);
 
-    const codeFromUrl = router.query.code;
-    redeemingCode(codeFromUrl);
+    redeemingCode(inputCode);
   };
 
   return ready && user ? (
```
Comment (Shreyaschorge) [inline pages/profile/redeem/[code].tsx:117]: After cancelling the error `inputCode` value remains `false`, resulting into a blank screen

image

image

**Review by Shreyaschorge — CHANGES_REQUESTED** [review]
Just a minor change, the rest seems fine.

File: pages/profile/redeem/[code].tsx (hunk 1 above)
Comment (sunilsabatp) [reply pages/profile/redeem/[code].tsx:117]: > After cancelling the error `inputCode` value remains `false`, resulting into a blank screen
> 
> image
> 
> image

@mariahosfeld what should I do in this case ?

Comment (sunilsabatp) [reply pages/profile/redeem/[code].tsx:117]: > After cancelling the error `inputCode` value remains `false`, resulting into a blank screen
> 
> image
> 
> image

Even not clicking to error  cancel button, the model disappear after some time , I'm not sure, Is this  a default behaviour or need to do some changes in it

**Review by Shreyaschorge — APPROVED** [review]
LGTM

**Pull request merged**